config/initializers/rswag_ui.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+Rswag::Ui.configure do |c|
```

Comment:
- [ ] You have some  pending tests, if you do not plan to use them kindly remove them

Reply:
I have deleted the redundant test files as you suggested.